source/api_documentation/network_integration/advertiser_campaigns/index.rst

```diff
@@ -259,6 +259,8 @@ Node Parameters and Usage
   * - Menu
     - \*prompt
 
+      asr_phrases
```

Comment:
The demo shows that `asr_phrases` is an optional parameter for all node types, but I only see it here for `Menu`. Should this also be under the other node types' parameters?

Reply:
I totally forgot to push my changes up 🤦‍♀️  they're here now!